.github/workflows/nextjs.yml

```diff
@@ -27,7 +27,6 @@ concurrency:
 jobs:
   # Build job
   build:
-    if: ((github.repository == vars.REPOSITORY) || (github.repository == 'bcgov/aries-oca-explorer'))
```

Comment:
Why did we remove this? This is set specifically to limit the execution of the job to PRs opened from within this repo/org and avoid PRs from forks to run this.